src/levenshtein.cr

```diff
@@ -10,60 +12,36 @@ module Levenshtein
   # Levenshtein.distance("こんにちは", "こんちは")           # => 1
   # Levenshtein.distance("hey", "hey")              # => 0
   # ```
-  def self.distance(string1 : String, string2 : String) : Int32
+  # If *cutoff* is given then the method is allowed to end once the loweset
+  # possible bound is greater than *cutoff* and return that lower bound.
```

Comment:
Could a doc/code example for this be given?

Alternatively, I think it's totally fine to add the `cutoff` logic in a separate PR. Given that it's a new feature, it could sparkle more discussion, eventually leading to this PR being merged less likely or less faster to happen (just a guess! I'm not actually requesting this)

Reply:
Good point. I don't think so, but also, the other PR can come/appear after this one is merged.

Reply:
I could also make a new pull request and first squash all the features together. Whatever is easier for you guys.